Framework/script/RepoCleaner/qcrepocleaner/config.yaml

```diff
@@ -25,9 +25,9 @@ Rules:
 #  - object_path: Test
 #    delay: 240
 #    policy: none_kept
-#  - object_path: .*
-#    delay: 1440
-#    policy: skip
+  - object_path: qc/DAQ/MO/daqTask/numberRDHs
+    delay: 1440
+    policy: none_kept
```

Comment:
did you enable this only for a test or this is actually needed?

Reply:
good catch, that was just for a test, I will remove it